doc/source/fkf_bo_hu.md

```diff
@@ -0,0 +1,29 @@
+# FKF Budaörs
+
+Support for schedules provided by [FKF Budaörs](https://www.fkf.hu/hulladeknaptar-budaors), serving Budaörs, HU.
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+    sources:
+    - name: fkf_bo_hu
+      args:
+        street: FULL_STREET_NAME
+```
+
+### Configuration Variables
+
+**STREET**
+*(string) (required)*
+
+
+## Example
+
+```yaml
+waste_collection_schedule:
+    sources:
+    - name: allerdale_gov_uk
```

Comment:
Wrong name (sorry didn't noticed earlier)

Reply:
I fixed it, thanks for noticing